cognite/neat/_session/_template.py

```diff
@@ -175,17 +175,26 @@ def data_product_model(
             self._state.last_reference = last_rules
         return issues
 
-    def conceptual_model(self, io: Any) -> None:
+    def conceptual_model(
+        self,
+        io: Any,
+        base_model: Literal["CogniteCore"] = "CogniteCore",
+        no_concepts: int | None = None,
```

Comment:
I read this as No concepts. What about?
```suggestion
        concept_count: int | None = None,
```

Reply:
renamed to total_concepts